portfolio/src/main/webapp/index.html

```diff
@@ -121,11 +121,13 @@
             </tr>
         </table>
     </div>
-    <form action="/data" method="POST">
+    <!-- POST request for comment below -->
+    <form action="/data" method="POST" onsubmit="return emptyBox()">
         <div class="comments">
-            <textarea name="comment-box"></textarea>
+            <textarea name="comment-box" id="comments-box"></textarea>
             <input type="submit"/>
         </div>
+        <!-- Comments displayed below -->
         <div id="display-comments">
```

Comment:
I think you misunderstood my comment here from the last PR –

This element: `<div id="display-comments"></div>`

Does not interact with the `<form>` element that encloses it.

So it would be best to keep it out of the form, i.e.

```
<form>
... form related stuff ...
</form>
<!-- Comments displayed below -->
<div id="display-comments">
</div>
```

Does that make sense?

If you rely on the div being within the form for styling purposes, you can probably achieve the same style effect with the div outside of the form by applying the css styles a bit differently. It's just a lot clearer to a reader of the code to only have things within a `<form>` element if they are actually used by the form, since there's a lot of interaction that goes on with that type of element.

Reply:
ohhh, i see. yeah, that makes sense. ill moved it to outside the form.